spark/src/main/scala/org/apache/spark/sql/delta/commands/DeleteWithDeletionVectorsHelper.scala

```diff
@@ -181,8 +181,20 @@ object DeleteWithDeletionVectorsHelper extends DeltaCommand {
     val (dvAddFiles, dvRemoveFiles) = dvUpdates.unzip
     val dvAddFilesWithStats = getActionsWithStats(spark, dvAddFiles, snapshot)
 
-    // TODO: gather more metrics
-    val metricMap = Map("numDeletedRows" -> numDeletedRows, "numRemovedFiles" -> numRemovedFiles)
+    var (numDeletionVectorsAdded: Long, numDeletionVectorsRemoved: Long
+    , numDeletionVectorsUpdated: Long) =
+      dvUpdates.foldLeft((0L, 0L, 0L)) {
+        case ((added, removed, updated), (t1, t2)) =>
```

Comment:
```suggestion
        case ((added, removed, updated), (addFile, removeFile)) =>
```
Makes the rest of the code easier to read.